apps/ledger/src/schema/repositories/schema.repository.ts

```diff
@@ -456,4 +459,35 @@ export class SchemaRepository {
       throw error;
     }
   }
+
+  async updateSchema(schemaDetails: IUpdateSchema): Promise<UpdateSchemaResponse> {
+    const { alias, schemaId, orgId } = schemaDetails;
+
+    try {
+      if (orgId) {
```

Comment:
Optimize this and remove the existence check, you can handle in query itself, like below :
{
  where: {
    schemaLedgerId: schemaId,
    ...(orgId ? { orgId } : {}),
  },
}